cpp/src/parquet/arrow/schema.h

```diff
@@ -51,44 +52,53 @@ ::arrow::Status NodeToField(const schema::Node& node,
 /// \param column_indices indices of leaf nodes in parquet schema tree. Appearing ordering
 ///                       matters for the converted schema. Repeated indices are ignored
 ///                       except for the first one
+/// \param properties reader options for FileReader
```

Comment:
It looks like the `properties` parameter is unused - is there a reason to prefer adding it?

Reply:
Re: above, the way that dictionary-encoded fields are handled in reader.cc right now is very hacky, I started adding logic to schema.cc to create DictionaryType instances and found that I was blocked by the current structure of reader.cc. I will return to this